### ISSUE TITLE
Potential problems when separating from swefreq

### ISSUE THREAD

**Issue by MalinAhlberg** [NBISweden/beacon-python#2]
#### Description:

A bunch of comments regarding the new migration script.

- regarding `variantType`, the values we have now are based on the lenght difference of the `ref` and the `alt` (https://github.com/NBISweden/swefreq/blob/develop/sql/beacon_schema.sql#L74) . I'm not sure we would like to keep it this way, and if not, if this is a good time to change it? @viklund ?
- in the current setup, with the beacon&swefreq data managed together, many versions of each dataset is kept, and filtered on based on the `available_from` field (https://github.com/NBISweden/swefreq/blob/develop/sql/beacon_schema.sql#L33). If we separate the data, should we keep something like that, or do we expect to only have one version of each dataset?
- Should we still keep the setup as in `hertz-config`? If so, should the migration script for the data be moved there? 

#### DoD (Definition of Done)
For each list item, either decide that we should ignore it, or create a new issue on how to move forward.

#### Testing
- Test querying the beacon: I guess the index provided by the [`init.sql`](https://github.com/NBISweden/beacon-python/blob/develop/data/init.sql#L69) should be enough. We should probably test some range queries before we start using this in production, though.

**Comment by viklund** [comment]
* `variantType`. Right now I'm leaning towards just going straight up with the CSC database schema, whatever that is, I guess it's a column there?
* Regarding available from. There are support for versions in the schema and I think we should use that for the multiple versions. But, it's probably not a bad idea to keep the available from thing in the schema somehow, it's quite useful to be able to put stuff in the db without it showing up online. If one were to take the CSC db schema and add this, would it be a lot of work or little work. If it's a very simple clean patch then I have no problems with it. If it becomes messy we should just skip it.
* I'm not really sure what you mean here.

**Comment by andkaha** [comment]
I'll take care of the first two points during the week. I was using the NBIS fork of the schema when creating the migration script, and I need to look at the CSCfi schema to make sure it's up to date with that now.

I will add an extra column for `available_from`. This will be a patch both to the migration script and to the `init.sql` file.

**Comment by MalinAhlberg** [comment]
@kusalananda ok, good!
Here are some clarification based on @viklund's questions and comments.
1. They do have the `variantType` in their schema, my question was more about the content of this field. CSC uses `cyvcf2` for the computing variantType, while we look at the length difference between `alt` and `ref`. It might be that what we're doing is perfectly fine, but since I'm not sure I wanted to raise the question (it could also be that we will want to change this later on, and keep everything as is for now).
3. What I meant was that maybe we shouldn't be using this repo, after all, but instead keep everything in the `hertz-config` repo? So far we have all our configs and modification there, and this repo has been working mirror of the CSC code, with the only addition being tags to remember which version we have in production.
